lib/parsers/parseOCLC.py

```diff
@@ -186,7 +197,7 @@ def extractHoldingsLinks(holdings, instance):
 
 
 def _loadURIIdentifier(uri):
-    uriGroup = re.search(r'\/((?:(?!\/).)+)$', uri)
+    uriGroup = re.search(r'\/((?:(?!\/)[^.])+(?=$|\.[a-z]{3,4}$))', uri)
```

Comment:
nit: I'd put a comment here on what this is doing so that friends in the future (and present) know what it's doing

Reply:
Good idea, I just had to look at it for a few minutes to remember exactly what it was doing